### ISSUE TITLE
156 line in train_ABP.py may be error

### ISSUE THREAD

**Issue by lizihao6868** [EthanZhu90/ZSL_ABP#4]
loss = opt.langevin_s*2/2 * loss may be error,loss = opt.langevin_s**2 / 2 * loss
